delete_records_request.go

```diff
@@ -74,13 +75,17 @@ func (d *DeleteRecordsRequest) key() int16 {
 }
 
 func (d *DeleteRecordsRequest) version() int16 {
-	return 0
+	return d.Version
 }
 
 func (d *DeleteRecordsRequest) headerVersion() int16 {
 	return 1
 }
 
+func (d *DeleteRecordsRequest) isValidVersion() bool {
+	return d.Version == 0
```

Comment:
Version 1 is just response throttling behaviour change so I think we support that too.